docs-client/src/containers/EnumPage/index.tsx

```diff
@@ -72,6 +73,7 @@ export default class EnumPage extends React.PureComponent<Props> {
                     <TableCell>
                       <code>{value.name}</code>
                     </TableCell>
+                    <TableCell>{value.literalValue}</TableCell>
```

Comment:
@trustin 
I google it and I found out jsx ignore booleans, null, undefined.
so It shows empty table cell
If there's good idea to alter empty cell, let me know. I'll work on it(maybe it would be another issue)
https://reactjs.org/docs/jsx-in-depth.html

Reply:
So we should show the value when it exists as we do in https://github.com/line/armeria/blob/master/docs-client/src/components/VariableList/index.tsx#L62